src/Stratis.Bitcoin/Connection/AddNodeRpcResult.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+
+namespace Stratis.Bitcoin.Connection
+{
+    /// <summary>
+    /// A model that represents the result of calling the <see cref="ConnectionManagerController.AddNodeApiAsync(string, string)"/> API method.
+    /// </summary>
+    public sealed class AddNodeRpcResult
+    {
+        /// <summary> Reports any error that might have occurred during the call.</summary>
+        [JsonProperty("errorMessage")]
+        public string ErrorMessage { get; set; }
+
+        /// <summary> Indicates that the add node call succeeeded.</summary>
+        [JsonProperty("success")]
+        public bool Success { get; set; }
```

Comment:
We typically don't have this pattern of returning a boolean success flag into the response.
The right pattern for RPC calls is to either return the result or throw an `RPCServerException` with the error message inside it.
The problem is that this `AddNode` method is in `Stratis.Bitcoin` and the `RPCServerException` type is in `Stratis.Bitcoin.Features.RPC`.
Can you please look into whether we could move the `ConnectionManagerController`'s RPC methods into  `Stratis.Bitcoin.Features.RPC`?